docs/packages/pkg/HiRedis.rst

```diff
@@ -0,0 +1,22 @@
+.. spelling::
+
+    HiRedis
+    hiredis
+
+.. index:: HiRedis
```

Comment:
please add the package to at least one of the already listed index-groups

https://hunter.readthedocs.io/en/latest/genindex.html